app/components/custom-modal/custom-modal.tsx

```diff
@@ -130,8 +142,8 @@ const useStyles = makeStyles(({ colors }, props: UseStylesProps) => ({
     fontSize: 24,
     fontWeight: Platform.OS === "ios" ? "600" : "700",
     lineHeight: 32,
-    maxWidth: "80%",
-    textAlign: "center",
+    maxWidth: props.titleMaxWidth || "80%",
+    textAlign: props.titleTextAlignment || "center",
```

Comment:
I don't think we need to override this? why do you think that is necessary?

having consistency across screen is important, so we would need a good reason to change the pattern